gradle/validation/owasp-dependency-check.gradle

```diff
@@ -26,9 +26,12 @@ configure(rootProject) {
   dependencyCheck {
     failBuildOnCVSS = propertyOrDefault("validation.owasp.threshold", 7) as Integer
     formats = ['ALL']
-    skipProjects = [':solr:solr-ref-guide']
+    skipProjects = [':solr:solr-ref-guide', ':solr-missing-doclet']
```

Comment:
`solr-missing-doclet` is a build only dependency and keeps getting flagged as a Solr 1.0 version with lots of CVEs - this doesn't need to be checked.